lib/tasks/import_staticstats.rake

```diff
@@ -0,0 +1,93 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task :staticstats, [:csv_file] => [:environment] do |t, args|
+
+    habitats = Habitat.all
+    
+    habitats.each do |habitat|
+      csv_file = "#{habitat.name.capitalize}_Protected.csv"
+      Rails.logger.info "import CSV: #{csv_file}"
+      import_csv_file(habitat.name, csv_file)
+      
+      csv_file = "#{habitat.name.capitalize}_Total.csv"
+      Rails.logger.info "import CSV: #{csv_file}"
+      import_csv_file(habitat.name, csv_file)     
+    end
+
+    puts "recalculate protected percentages"
+    recalculate_protected_percentages
+
+  end
+  
+  def import_csv_file(habitat, csv_file)
+    filename = "#{Rails.root}/lib/data/#{csv_file}"
+    
+    csv = File.open(filename, encoding: "utf-8")
+    csv_headers = File.readlines(csv).first.split(",")
+    
+    staticstats_hash = {
+      iso3: csv_headers[0],
+      value: csv_headers[1].chomp
+    }
+    
+    CSV.parse(csv, headers: true, encoding: "utf-8") do |row|
+      csv_staticstats_row = row.to_hash
+      staticstats_row_hash = {}
+      iso3 = ""
+      value = 0
+
+      staticstats_hash.keys.each do |key|
+        if key == :iso3
+          iso3 = csv_staticstats_row[staticstats_hash[key]]&.strip
+        elsif key == :value
+          value = csv_staticstats_row[staticstats_hash[key]]&.strip
+        end
+      end
+      
+      if csv_file.include? "Protected"
+        insert_static_stat(csv_file, "protected", habitat, iso3, value)
+      elsif csv_file.include? "Total"
+        insert_static_stat(csv_file, "total", habitat, iso3, value)
+      end
+    end
+      
+    csv.close
+    
+  end
+  
+  def insert_static_stat(csv_file, kind, habitat, iso3, value)
+    return if iso3.include? "/"
+    return if iso3.include? "ABNJ"
```

Comment:
I think here you could do so something like
`return if ['ABNJ', '/'].include?(iso3)

Reply:
Oh nice! Thanks @lucacug

Reply:
Sorry, Andrew in this case I gave you a bad hint.
in this case, the `include?` for the array doesn't work because it checks the exact match, so you have to do something like:
`return if iso3.include? '/' || iso3.include? 'ABNJ' `
Sorry but I just discovered that the method `include?` has two different behaviours if  it deals with an array or string

Reply:
No worries I will take a look!